mongo_orchestration/common.py

```diff
@@ -80,19 +84,36 @@ def _strip_auth(self, proc_params):
         params.pop("clusterAuthMode", None)
         return params
 
-    def mongodb_auth_uri(self, hosts):
-        """Get a connection string with all info necessary to authenticate."""
-        parts = ['mongodb://']
+    def mongodb_uri(self, hosts, uri_opts):
+        """Append URI options for auth, TLS, etc."""
```

Comment:
Not sure if this kind of comment is appropriate or if this should use `#`. Please advise.